oidc-client-app-samples/browser-client-app-sample/app/profile.html

```diff
@@ -0,0 +1,172 @@
+                          <th>User's name</th>
+                          <th>Given name</th>
+                          <th>Family name</th>
+                          <th>Preferred username</th>
+                          <th>Email</th>
+                          <th>Picture</th>
+                       </tr>
+                    </thead>
+                    <tbody>
+                       <tr>
+                          <td id="sub"></td>
+                          <td id="name"></td>
+                          <td id="given_name"></td>
+                          <td id="family_name"></td>
+                          <td id="preferred_username"></td>
+                          <td id="email"></td>
+                          <td id="picture_td"><a href="" id="picture" /></td>
+                       </tr>
+                    </tbody>
+                 </table>
+              </div>
+           </div>
+        </div>
+     </div>
+  </section>
+  <!-- Footer -->
+  <footer id="footer">
+     <div class="container">
+        <div class="row">
+           <div class="col-md-10 col-md-offset-1 text-center">
+              <span class="text-muted">Copyright &copy;  <a href="http://wso2.com/" target="_blank" class="wso2-logo-link"><img src="img/wso2-dark.svg" class="wso2-logo" alt="wso2-logo"/></a> &nbsp;2017</span>
+           </div>
+        </div>
+     </div>
+     <a id="to-top" href="#top" class="btn btn-dark btn-lg"><i class="fa fa-chevron-up fa-fw fa-1x"></i></a>
+  </footer>
+
+  <!-- extras -->
+  <!-- Bootstrap Core CSS -->
+  <link href="libs/bootstrap_3.3.7/css/bootstrap.min.css" rel="stylesheet">
+
+  <!-- Custom CSS -->
+  <link rel="stylesheet" href="./css/app.css" />
+
+  <!-- Custom Fonts -->
+  <link href="libs/font-awesome_4.3.0/css/font-awesome.min.css" rel="stylesheet" type="text/css">
+  <link href="css/font.css" rel="stylesheet" type="text/css">
+  <!-- scripts -->
+  <script src="js/jquery-3.2.1.min.js"></script>
+  <!-- Bootstrap Core JavaScript -->
+  <script src="libs/bootstrap_3.3.7/js/bootstrap.min.js"></script>
+  <script src="./js/bundle.js"></script>
+  <script type="text/javascript">
+
+    var application = null;
+    var init = function () {
+      application = new App({
+    'authorizeUrl': 'https://localhost:9443/oauth2/authorize',
+    'tokenUrl': 'https://localhost:9443/oauth2/token',
+    'revokeUrl': 'https://localhost:9443/oauth2/revoke',
+    'logoutUrl': 'https://localhost:9443/oidc/logout',
+    'userInfoUrl': 'https://localhost:9443/oauth2/userinfo',
+    'flowType': "PKCE",
+    'userStore': "LOCAL_STORAGE",
+    'clientId': '0Ob7gGlPWWoCMx4qrGNSWjRgnMwa',
+    'clientSecret': '4jjZU5nke4iovlq5seaSPVxnqBMa',
+    'redirectUri': 'http://localhost:8080/app/home.html',
+    'scope': 'openid',
+    'postLogoutRedirectUri': 'http://localhost:8080/app/'
+  });
+  application.init();
+      // check for authorization response if available.
+      application.checkForAuthorizationResponse();
```

Comment:
Let's explain in a comment, what changes needed for  displaying additional claims

Reply:
done